styles.css

```diff
@@ -0,0 +1,409 @@
+/*        O__O
+          (oo)
+   /-------\/
+  / |     ||
+ *  ||----||
+    ~~    ~~
+Cow at Disneyland
+*/
+
+
+/* MASTER STYLES */
+* {
+	font-family: din-condensed, sans-serif;
+	font-weight: 400;
+	font-style: normal;
+}
+
+body {
+	margin: 0;
+}
+
+.container {
+	width: 85%;
+	margin: 0 auto;
+}
+
+a {
+	text-decoration: none;
+	color: #ffffff;
+}
+
+a:hover {
+	color: #929292;
+}
+
+i {
+	color: #000000;
+	font-size: 40px;
+}
+
+/*          (__)
+            (@@)
+     /-------\/
+    / |     ||
+   *  ||----||
+      ^^    ^^
+   Cow who ate
+psychadelic mushrooms*/
+
+/*NAVBAR STYLES*/
+
+nav {
+	overflow: auto;
+	background-color: #ffffff;
+	color: #000000;
+	font-size: 25px;
+	height: 70px;
+	}
+
+nav #logo {
+	float: left;
+	font-size: 40px;
+	padding: 8px;
+	font-weight: 600;
+	margin-left: 30px;
+	margin-top: 8px;
+}
+
+#hamburger {
+	float: right;
+	display: none;
+}
+
+.hamburgerRow {
+ 	width: 30px;
+ 	height: 4px;
+ 	background-color: black;
+ 	margin: 6px 0;
+}
+
+nav ul li {
+	float: right;
+	margin-right: 60px;
+	list-style: none;
+}
+
+nav ul li a {
+	color: #000000;
+	padding: 7px;
+	padding-bottom: 0px;
+	font-weight: 600;
+}
+
+nav ul li a:hover {
+	border: .5px solid #000000;
+	border-radius: 5px;
+	padding: 6.5px;
+}
+
+/*             )\               (__)
+              /  \              (oo)
+        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+           Cow swimming at Amity Island
+(Where Jaws was set, for those less educated)*/
+
+/* HERO STYLES */
+
+.hero {
+	margin: 0;
+	color: #ffffff;
+	background-color: #000000;
+	background-image: url(images/hero.jpg);
+	background-position: center;
+  	background-repeat: no-repeat;
+  	background-size: cover;
+  	position: relative;
+	height: 870px;
+}
+
+#hero-text {
+	margin: 0 0 0 20px;
+	font-size: 46px;
+	font-family: 'Lora', serif;
+	position: relative;
+	top: 550px;
+	left: 50px; 
+}
+
+.hero h1 {
+	margin: 0;
+	font-family: 'Lora', serif;
+}
+
+.hero h3 {
+	margin: 0;
+	font-size: 95%;
+}
+
+.hero h6 {
+	margin: 0;
+	font-size: 40%;
+}
+
+#hero-arrow {
+	display: inline-block;
+	position: relative;
+	text-decoration: none;
+	color: #ffffff;
```

Comment:
not doing anything

Reply:
hero arrow removed anyway due to dubious position